### DISCUSSION TITLE
3d printing a hoarding model in memory of a friend

### DISCUSSION THREAD

**Discussion by RangerElite17** [sn4k3/UVtools#825] [Q&A]
My friend got a 3d printer over the holdiays but passed before he could use it. I made a hoarding bug in blender and got the file to a cxdlp and tried it in the printer (Halot mage). It didn't work, but I changed it to a .cxdlpv4 and it said it was the wrong resolution. The file is in 1620 but needs to be in 8k I think. Is there a way I can do that?

**Comment by sn4k3** [comment]
You need to slice your model(stl) with the correct printer on a capable slicer. In your case and if printer is the Halot mage, you need to select it on slicer so it output the correct file.
